Doc/library/dataclasses.rst

```diff
@@ -182,24 +182,24 @@ Module contents
 
     .. versionadded:: 3.10
 
-   - ``slots``: If true (the default is ``False``), :attr:`~object.__slots__` attribute
+   - *slots*: If true (the default is ``False``), :attr:`~object.__slots__` attribute
      will be generated and new class will be returned instead of the original one.
      If :attr:`!__slots__` is already defined in the class, then :exc:`TypeError`
      is raised.
 
     .. versionadded:: 3.10
 
     .. versionchanged:: 3.11
-       If a field name is already included in the ``__slots__``
-       of a base class, it will not be included in the generated ``__slots__``
+       If a field name is already included in the :attr:`!__slots__`
```

Comment:
```suggestion
       If a field name is already included in the :attr:`~object.__slots__`
```

Reply:
I'm not sure this one is needed; we already have the link in L185, and this is the `versionchanged` for that paragraph.